research/blueprint_algo/blueprint_short_deck_poker.py

```diff
@@ -71,15 +76,18 @@ def update_strategy(state: ShortDeckPokerState, i: int):
     """
     ph = state.player_i  # this is always the case no matter what i is
 
-    if state.is_terminal or state.players[i].is_active is False:
-        # or if betting round is > 0, strategy is only
-        # updated in betting round 1 for Pluribus, but I am doing all rounds in this example
+    player_not_in_hand = not state.players[i].is_active
+    if state.is_terminal or player_not_in_hand or state.betting_round > 0:
         return
+    # NOTE(fedden): According to Algorithm 1 in the supplementary material,
```

Comment:
Yes that sounds good. Will need to rework things a little!

Reply:
But that can be done in another PR. 

Otherwise the later part is a little more challenging logically than the quick review I'm giving, but it reads like it makes sense. I think we should merge and I'll start stepping through the game play.

Reply:
awesome thanks!